.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image
+      run: docker build alpine/. --file Dockerfile --tag my-image-name:$(date +%s)
```

Comment:
Do you know if github CI can run a script ? As I've a build hook script for Docker hub may be we can use it ? At least Ci will test this script too ;-)

Thinking of something like: `(cd alpine && bash hook/build)`

Reply:
That looks like it works.

I'm a bit new to Github actions but it seems to work in my fork so far.

I doubt it will be a problem but GitHub provide 2,000 actions minutes/month and based on my testing each CI build takes between 8 and 9 minutes, so there should be enough quota to run over 200 CI builds/tests per month.